src/styles/css/hero.css

```diff
@@ -0,0 +1,60 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500&display=swap');
```

Comment:
you do not need to reimport the font, it is already imported in the index.css
this line of code should be removed